xtask/src/commands/devtools_test.rs

```diff
@@ -2,18 +2,28 @@ use std::error::Error;
 
 use xshell::{cmd, Shell};
 
-use crate::extendrtests::with_absolute_path::{swap_extendr_api_path, R_FOLDER_PATH};
+use crate::{
+    cli::devtools_test::DevtoolsTestArg,
+    extendrtests::with_absolute_path::{swap_extendr_api_path, R_FOLDER_PATH},
+};
 
-pub(crate) fn run(shell: &Shell) -> Result<(), Box<dyn Error>> {
+pub(crate) fn run(shell: &Shell, args: DevtoolsTestArg) -> Result<(), Box<dyn Error>> {
     let _document_handle = swap_extendr_api_path(shell)?;
 
-    run_tests(shell)?;
+    run_tests(shell, args)?;
 
     Ok(())
 }
 
-fn run_tests(shell: &Shell) -> Result<(), Box<dyn Error>> {
+fn run_tests(shell: &Shell, args: DevtoolsTestArg) -> Result<(), Box<dyn Error>> {
     let _r_path = shell.push_dir(R_FOLDER_PATH);
+    if args.snapshot_accept {
+        cmd!(
+            shell,
+            "Rscript -e testthat::snapshot_accept(\"macro-snapshot\")"
```

Comment:
I am not sure this will work because regular devtools test runs in a temp directory so new snapshots are not recorded in the project folder. Did you check it out?

Reply:
Ehh, good question. I've been using it for a while now. But I might be wrong.

Reply:
Alright, I just tested this, and it does work.
I added
```rs
        const _:() = {
            
        };
```
to extendr_function.rs. Calling `cargo extendr devtools-test`, then a `tests/extendrtests/tests/testthat/_snaps/macro-snapshot.new.md` is generated. This is then accepted `cargo extendr devtools-test -a` and a new committable version is ready to go.